gpgme.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "gpgme",
```

Comment:
this is in the runtime.

Reply:
Or at least the dependencies since it seems that gpgme is missing Qt support in the runtime.

Reply:
@hfiguiere 
gpgme seems to be required. Build fails without that.